package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "engines": {
     "node": "10.15.3",
-    "yarn": "1.13.0"
+    "yarn": "1.15.2"
```

Comment:
we just discussed to specify the yarn version as `^1.15.2` to avoid further breakdowns of CI because of Xenial distro upgrades